src/components/dialogs/network-modifications/line/modification/line-modification-dialog.js

```diff
@@ -28,7 +28,7 @@ import {
 } from 'components/utils/field-constants';
 import { FormProvider, useForm } from 'react-hook-form';
 import { sanitizeString } from 'components/dialogs/dialogUtils';
-import { microUnitToUnit, unitToMicroUnit } from 'utils/rounding';
+import { microUnitToUnit, unitToMicroUnit } from 'utils/unit-converter.tsx';
```

Comment:
why .tsx in import ? all the others are 'utils/unit-converter'

Reply:
Useless indeed. `.tsx` removed from 3 files.